Index: Utility/DbDefaultConfiguration.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\n$host = \"localhost\";\r\n$database = \"livent\";\r\n$username = \"root\";\r\n$password = \"\";\r\n\r\n$data=new FDb($host,$database,$username,$password);\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Utility/DbDefaultConfiguration.php b/Utility/DbDefaultConfiguration.php
--- a/Utility/DbDefaultConfiguration.php	(revision a518d72e7c15693b9213e2e749e2041b6087cf09)
+++ b/Utility/DbDefaultConfiguration.php	(date 1657548370670)
@@ -3,6 +3,6 @@
 $host = "localhost";
 $database = "livent";
 $username = "root";
-$password = "";
+$password = "pippo";
 
 $data=new FDb($host,$database,$username,$password);
